src/api/chainHead.md

```diff
@@ -14,7 +14,7 @@ This beginner guide shows how to use the `chainHead` functions in order to know
 
 1. Call `chainHead_unstable_follow` with `withRuntime: true` to obtain a `followSubscription`. This `followSubscription` will need to be passed when calling most of the other `chainHead`-prefixed functions. If at any point in the future the JSON-RPC server sends back a `{"event": "stop"}` notification, jump back to step 1.
 
-2. When the JSON-RPC server sends back a `{"event": "initialized"}` notification with `subscription` equal to your `followSubscription`, store the value of `finalizedBlockHashes` found in that notification. If `finalizedBlockHashes` contains multiple values, the user is encouraged to use the last one. This is the hash of the last block that has been finalized, for the purpose of this guide it will be named the `currentFinalizedBlock`. For all other values in `finalizedBlockHashes`, the user is encouraged to call `chainHead_unstable_unpin` with the `followSubscription`.
+2. When the JSON-RPC server sends back a `{"event": "initialized"}` notification with `subscription` equal to your `followSubscription`, store the value of `finalizedBlockHashes` found in that notification. If `finalizedBlockHashes` contains multiple values, the user is encouraged to use the last one. This is the hash of the last block that has been finalized, and for the purpose of this guide we'll refer to it as the _current finalized block_. For all other values in `finalizedBlockHashes`, the user is encouraged to call `chainHead_unstable_unpin` with the `followSubscription`.
```

Comment:
Remember to update the other mentions of `currentFinalizedBlock` and also iirc one saying "last finalized block" :)